Automattic/delphin
# Add colors

This pull request add colors because without colors the world would be a boring place. More specifically it adds a CSS reset and styles for the `Home` page:

![screenshot](https://cloud.githubusercontent.com/assets/594356/14792339/12af0626-0b1a-11e6-8136-f414b9c13381.png)
#### Testing instructions
1. Run `git checkout add/colors` and start your server
2. Open the [`Home` page](http://localhost:1337/)
3. Check that your eyes are no longer bleeding
#### Reviews
